### DISCUSSION TITLE
Libre 2 USA Android app released now what???

### DISCUSSION THREAD

**Discussion by stlrivals** [Navid200/xDrip#137] [Development]
When do we anticipate support for the Libre 2 USA?

**Comment by Navid200** [comment]
We have discussions now opened in the main repository:
https://github.com/NightscoutFoundation/xDrip/discussions

I am looking for a way to transfer the open discussions from here to there. But, I'm not sure I'll succeed. In that case, it may be easier to open new discussions there.
